docs/docs/.vuepress/config.js

```diff
@@ -18,195 +19,8 @@ module.exports = {
-          "/workflows/steps/triggers/",
-          "/workflows/events/",
-          "/components/actions/",
-          "/workflows/concurrency-and-throttling/",
-          "/environment-variables/",
-          "/workflows/settings/",
-          "/workflows/networking/",
-          "/migrate-from-v1/",
-        ]
-      },
-      "/sources/",
-      "/connected-accounts/",
-      "/user-settings/",
-      {
-        title: "Reference: Code, APIs + CLI",
-        children: [
-          {
-            title: "Writing Code in Workflows",
-            type: "group",
-            initialOpenGroupIndex: 1,
-            children: [
-              "/code/",
-              {
-                title: "Node.js",
-                type: "group",
-                children: [
-                  "/code/nodejs/",
-                  "/code/nodejs/auth/",
-                  "/code/nodejs/http-requests/",
-                  "/code/nodejs/working-with-files/",
-                  "/code/nodejs/async/",
-                ],
-              },
-              "/code/python/",
-              "/code/go/",
-              "/code/bash/",
-              "/destinations/",
-              "/scheduling-future-tasks/",
-            ],
-          },
-          {
-            title: "Authoring Components",
-            type: "group",
-            children: [
-              "/components/",
-              "/components/quickstart/nodejs/actions/",
-              "/components/quickstart/nodejs/sources/",
-              "/components/api/",
-              "/components/guidelines/",
-              "/pipedream-axios/",
-            ],
-          },
-          {
-            title: "CLI",
-            type: "group",
-            children: ["/cli/install/", "/cli/login/", "/cli/reference/"],
-          },
-          {
-            title: "APIs",
-            type: "group",
-            children: [
-              "/api/overview/",
-              "/api/auth/",
-              "/api/rest/",
-              "/api/rest/webhooks/",
-              "/api/rest/rss/",
-              "/api/rest/workflow-errors/",
-              "/api/sse/",
-            ],
-          },
-          {
-            title: "Integrations",
-            type: "group",
-            children: [
-              "/apps/all-apps/",
-              "/apps/discord/",
-              "/apps/hubspot/",
-              "/apps/servicenow/",
-              "/apps/slack/",
-              "/apps/strava/",
-              "/apps/twitter/",
-              "/apps/zoom/",
-            ],
-          },
-        ],
-      },
-      "/troubleshooting/",
-      {
-        title: "Privacy & Security",
-        children: [
-          "/privacy-and-security/",
-          "/privacy-and-security/best-practices/",
-          "/abuse/",
-          "/privacy-and-security/pgp-key/",
-          "/subprocessors/",
-        ],
-      },
-      {
-        title: "Pricing & Limits",
-        collapsable: true,
-        children: [
-          "/limits/",
```

Comment:
The limits and cold starts docs appear to have been removed in the transition. I'd still like to keep all these somewhere easy to find, even if it's in an Advanced / Reference section.

Reply:
Where do you think these should go in the new nav?